docs/how-to/installation/aws.txt

```diff
@@ -286,6 +286,36 @@ Listing Clusters
       -  AWS profile used for deploying cluster resources.
       -  default
 
+.. _aws-modifying-deployment:
+
+************************
+ Modifying a Deployment
+************************
+
+To modify an already deployed cluster you have a few options:
+
+1. If what you'd like to change is provided as a ``det deploy`` CLI
+option, you can re-deploy the cluster using ``det deploy``. Simply
+provide the updated option while keeping the ``cluster-id`` the same.
```

Comment:
I think it's worth mentioning they need to provide *all* original options, as well as the updated ones? Something like:

"Use the same full ``det deploy`` command as on cluster creation, but update the options as necessary, while keeping the ``cluster-id`` the same."